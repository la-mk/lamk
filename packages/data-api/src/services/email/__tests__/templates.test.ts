import { getEmailTemplate } from '../templateProcessor';

describe('email templates', () => {
  it('can render a reset password template', async () => {
    const resetEmail = await getEmailTemplate('reset-password', {
      storeName: 'Demo',
      storeUrl: 'https://demo.la.mk',
      resetLink: 'https://demo.la.mk/account/resetPassword?resetToken=23456',
    });

    expect(resetEmail.includes('https://demo.la.mk')).toBeTruthy();
    expect(resetEmail).toMatchSnapshot();
  });
});
