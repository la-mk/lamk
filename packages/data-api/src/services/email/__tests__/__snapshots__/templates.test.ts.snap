// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`email templates can render a reset password template 1`] = `
"
    <!doctype html>
    <html xmlns=\\"http://www.w3.org/1999/xhtml\\" xmlns:v=\\"urn:schemas-microsoft-com:vml\\" xmlns:o=\\"urn:schemas-microsoft-com:office:office\\">
      <head>
        <title>
          Reset password
        </title>
        <!--[if !mso]><!-- -->
        <meta http-equiv=\\"X-UA-Compatible\\" content=\\"IE=edge\\">
        <!--<![endif]-->
        <meta http-equiv=\\"Content-Type\\" content=\\"text/html; charset=UTF-8\\">
        <meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1\\">
        <style type=\\"text/css\\">
          #outlook a { padding:0; }
          body { margin:0;padding:0;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%; }
          table, td { border-collapse:collapse;mso-table-lspace:0pt;mso-table-rspace:0pt; }
          img { border:0;height:auto;line-height:100%; outline:none;text-decoration:none;-ms-interpolation-mode:bicubic; }
          p { display:block;margin:13px 0; }
        </style>
        <!--[if mso]>
        <xml>
        <o:OfficeDocumentSettings>
          <o:AllowPNG/>
          <o:PixelsPerInch>96</o:PixelsPerInch>
        </o:OfficeDocumentSettings>
        </xml>
        <![endif]-->
        <!--[if lte mso 11]>
        <style type=\\"text/css\\">
          .mj-outlook-group-fix { width:100% !important; }
        </style>
        <![endif]-->
        
      <!--[if !mso]><!-->
        <link href=\\"https://fonts.googleapis.com/css?family=Ubuntu:300,400,500,700\\" rel=\\"stylesheet\\" type=\\"text/css\\">
        <style type=\\"text/css\\">
          @import url(https://fonts.googleapis.com/css?family=Ubuntu:300,400,500,700);
        </style>
      <!--<![endif]-->

    
        
    <style type=\\"text/css\\">
      @media only screen and (min-width:480px) {
        .mj-column-per-100 { width:100% !important; max-width: 100%; }
      }
    </style>
    
  
        <style type=\\"text/css\\">
        
        

    @media only screen and (max-width:480px) {
      table.mj-full-width-mobile { width: 100% !important; }
      td.mj-full-width-mobile { width: auto !important; }
    }
  
        </style>
        <style type=\\"text/css\\">body {
        background-color: #F6F8FF;
        font-family: \\"Ubuntu\\", Helvetica, Arial, sans-serif;
        font-size: 14px;
        padding: 0px;
        color: #404C4D;
      }

      a {
        text-decoration: none;
        color: inherit;
      }</style>
        
      </head>
      <body>
        
        
      <div
         style=\\"\\"
      >
        <!-- HEADER -->
      
      <!--[if mso | IE]>
      <table
         align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\"
      >
        <tr>
          <td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\">
      <![endif]-->
    
      
      <div  style=\\"margin:0px auto;max-width:600px;\\">
        
        <table
           align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"
        >
          <tbody>
            <tr>
              <td
                 style=\\"direction:ltr;font-size:0px;padding:16px 0;text-align:center;\\"
              >
                <!--[if mso | IE]>
                  <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
                
        <tr>
      
            <td
               class=\\"\\" style=\\"vertical-align:top;width:600px;\\"
            >
          <![endif]-->
            
      <div
         class=\\"mj-column-per-100 mj-outlook-group-fix\\" style=\\"font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"
      >
        
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" width=\\"100%\\"
      >
        <tbody>
          <tr>
            <td  style=\\"vertical-align:top;padding:0px;\\">
              
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"\\" width=\\"100%\\"
      >
        
            <tr>
              <td
                 align=\\"center\\" style=\\"font-size:0px;padding:0px;word-break:break-word;\\"
              >
                
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"border-collapse:collapse;border-spacing:0px;\\"
      >
        <tbody>
          <tr>
            <td  style=\\"width:142px;\\">
              
      <img
         alt=\\"logo\\" height=\\"56\\" src=\\"https://la.mk/logo-horizontal.svg\\" style=\\"border:0;display:block;outline:none;text-decoration:none;height:56px;width:100%;font-size:13px;\\" width=\\"142\\"
      />
    
            </td>
          </tr>
        </tbody>
      </table>
    
              </td>
            </tr>
          
      </table>
    
            </td>
          </tr>
        </tbody>
      </table>
    
      </div>
    
          <!--[if mso | IE]>
            </td>
          
        </tr>
      
                  </table>
                <![endif]-->
              </td>
            </tr>
          </tbody>
        </table>
        
      </div>
    
      
      <!--[if mso | IE]>
          </td>
        </tr>
      </table>
      
      <table
         align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\"
      >
        <tr>
          <td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\">
      <![endif]-->
    
      
      <div  style=\\"margin:0px auto;max-width:600px;\\">
        
        <table
           align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"
        >
          <tbody>
            <tr>
              <td
                 style=\\"direction:ltr;font-size:0px;padding:0px;text-align:center;\\"
              >
                <!--[if mso | IE]>
                  <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
                
        <tr>
      
            <td
               class=\\"\\" style=\\"vertical-align:top;width:600px;\\"
            >
          <![endif]-->
            
      <div
         class=\\"mj-column-per-100 mj-outlook-group-fix\\" style=\\"font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"
      >
        
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" width=\\"100%\\"
      >
        <tbody>
          <tr>
            <td  style=\\"vertical-align:top;padding:0px;\\">
              
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"\\" width=\\"100%\\"
      >
        
            <tr>
              <td
                 style=\\"font-size:0px;padding:0px;word-break:break-word;\\"
              >
                
      
    <!--[if mso | IE]>
    
        <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\"><tr><td height=\\"16\\" style=\\"vertical-align:top;height:16px;\\">
      
    <![endif]-->
  
      <div
         style=\\"height:16px;\\"
      >
        &nbsp;
      </div>
      
    <!--[if mso | IE]>
    
        </td></tr></table>
      
    <![endif]-->
  
    
              </td>
            </tr>
          
      </table>
    
            </td>
          </tr>
        </tbody>
      </table>
    
      </div>
    
          <!--[if mso | IE]>
            </td>
          
        </tr>
      
                  </table>
                <![endif]-->
              </td>
            </tr>
          </tbody>
        </table>
        
      </div>
    
      
      <!--[if mso | IE]>
          </td>
        </tr>
      </table>
      
      <table
         align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\"
      >
        <tr>
          <td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\">
      <![endif]-->
    
      
      <div  style=\\"background:#ffffff;background-color:#ffffff;margin:0px auto;max-width:600px;\\">
        
        <table
           align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"background:#ffffff;background-color:#ffffff;width:100%;\\"
        >
          <tbody>
            <tr>
              <td
                 style=\\"direction:ltr;font-size:0px;padding:32px;text-align:center;\\"
              >
                <!--[if mso | IE]>
                  <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
                
            <tr>
              <td
                 class=\\"\\" width=\\"600px\\"
              >
          
      <table
         align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:536px;\\" width=\\"536\\"
      >
        <tr>
          <td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\">
      <![endif]-->
    
      
      <div  style=\\"margin:0px auto;max-width:536px;\\">
        
        <table
           align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"
        >
          <tbody>
            <tr>
              <td
                 style=\\"direction:ltr;font-size:0px;padding:16px 0;text-align:center;\\"
              >
                <!--[if mso | IE]>
                  <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
                
        <tr>
      
            <td
               class=\\"\\" style=\\"vertical-align:top;width:536px;\\"
            >
          <![endif]-->
            
      <div
         class=\\"mj-column-per-100 mj-outlook-group-fix\\" style=\\"font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"
      >
        
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" width=\\"100%\\"
      >
        <tbody>
          <tr>
            <td  style=\\"vertical-align:top;padding:0px;\\">
              
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"\\" width=\\"100%\\"
      >
        
            <tr>
              <td
                 align=\\"center\\" style=\\"font-size:0px;padding:0px;word-break:break-word;\\"
              >
                
      <div
         style=\\"font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:28px;font-weight:300;line-height:1;text-align:center;color:#687C94;\\"
      >Forgotten password request</div>
    
              </td>
            </tr>
          
      </table>
    
            </td>
          </tr>
        </tbody>
      </table>
    
      </div>
    
          <!--[if mso | IE]>
            </td>
          
        </tr>
      
                  </table>
                <![endif]-->
              </td>
            </tr>
          </tbody>
        </table>
        
      </div>
    
      
      <!--[if mso | IE]>
          </td>
        </tr>
      </table>
      
              </td>
            </tr>
          
            <tr>
              <td
                 class=\\"\\" width=\\"600px\\"
              >
          
      <table
         align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:536px;\\" width=\\"536\\"
      >
        <tr>
          <td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\">
      <![endif]-->
    
      
      <div  style=\\"margin:0px auto;max-width:536px;\\">
        
        <table
           align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"
        >
          <tbody>
            <tr>
              <td
                 style=\\"direction:ltr;font-size:0px;padding:16px 0;text-align:center;\\"
              >
                <!--[if mso | IE]>
                  <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
                
        <tr>
      
            <td
               class=\\"\\" style=\\"vertical-align:top;width:536px;\\"
            >
          <![endif]-->
            
      <div
         class=\\"mj-column-per-100 mj-outlook-group-fix\\" style=\\"font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"
      >
        
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" width=\\"100%\\"
      >
        <tbody>
          <tr>
            <td  style=\\"vertical-align:top;padding:0px;\\">
              
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"\\" width=\\"100%\\"
      >
        
            <tr>
              <td
                 align=\\"center\\" style=\\"font-size:0px;padding:0px;word-break:break-word;\\"
              >
                
      <div
         style=\\"font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:16px;font-weight:300;line-height:150%;text-align:center;color:#070708;\\"
      >It seems you have requested to reset your password on <a href=\\"https://demo.la.mk\\" target=\\"_blank\\"><strong>Demo</strong></a>, a store powered by <a href=\\"https://la.mk\\" target=\\"_blank\\"><strong>la.mk</strong></a>.</div>
    
              </td>
            </tr>
          
      </table>
    
            </td>
          </tr>
        </tbody>
      </table>
    
      </div>
    
          <!--[if mso | IE]>
            </td>
          
        </tr>
      
                  </table>
                <![endif]-->
              </td>
            </tr>
          </tbody>
        </table>
        
      </div>
    
      
      <!--[if mso | IE]>
          </td>
        </tr>
      </table>
      
              </td>
            </tr>
          
            <tr>
              <td
                 class=\\"\\" width=\\"600px\\"
              >
          
      <table
         align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:536px;\\" width=\\"536\\"
      >
        <tr>
          <td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\">
      <![endif]-->
    
      
      <div  style=\\"margin:0px auto;max-width:536px;\\">
        
        <table
           align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"
        >
          <tbody>
            <tr>
              <td
                 style=\\"direction:ltr;font-size:0px;padding:0px;text-align:center;\\"
              >
                <!--[if mso | IE]>
                  <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
                
        <tr>
      
            <td
               class=\\"\\" style=\\"vertical-align:top;width:536px;\\"
            >
          <![endif]-->
            
      <div
         class=\\"mj-column-per-100 mj-outlook-group-fix\\" style=\\"font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"
      >
        
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" width=\\"100%\\"
      >
        <tbody>
          <tr>
            <td  style=\\"vertical-align:top;padding:0px;\\">
              
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"\\" width=\\"100%\\"
      >
        
            <tr>
              <td
                 align=\\"center\\" vertical-align=\\"middle\\" style=\\"font-size:0px;padding:32px 0 16px 0;word-break:break-word;\\"
              >
                
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"border-collapse:separate;line-height:100%;\\"
      >
        <tr>
          <td
             align=\\"center\\" bgcolor=\\"#EF4351\\" role=\\"presentation\\" style=\\"border:none;border-radius:3px;cursor:auto;height:36px;mso-padding-alt:10px 25px;background:#EF4351;\\" valign=\\"middle\\"
          >
            <a
               href=\\"https://demo.la.mk/account/resetPassword?resetToken=23456\\" style=\\"display:inline-block;background:#EF4351;color:#ffffff;font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:18px;font-weight:normal;line-height:120%;margin:0;text-decoration:none;text-transform:none;padding:10px 25px;mso-padding-alt:0px;border-radius:3px;\\" target=\\"_blank\\"
            >
              Reset your password
            </a>
          </td>
        </tr>
      </table>
    
              </td>
            </tr>
          
      </table>
    
            </td>
          </tr>
        </tbody>
      </table>
    
      </div>
    
          <!--[if mso | IE]>
            </td>
          
        </tr>
      
                  </table>
                <![endif]-->
              </td>
            </tr>
          </tbody>
        </table>
        
      </div>
    
      
      <!--[if mso | IE]>
          </td>
        </tr>
      </table>
      
              </td>
            </tr>
          
            <tr>
              <td
                 class=\\"\\" width=\\"600px\\"
              >
          
      <table
         align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:536px;\\" width=\\"536\\"
      >
        <tr>
          <td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\">
      <![endif]-->
    
      
      <div  style=\\"margin:0px auto;max-width:536px;\\">
        
        <table
           align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"
        >
          <tbody>
            <tr>
              <td
                 style=\\"direction:ltr;font-size:0px;padding:0px;text-align:center;\\"
              >
                <!--[if mso | IE]>
                  <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
                
        <tr>
      
            <td
               class=\\"\\" style=\\"vertical-align:top;width:536px;\\"
            >
          <![endif]-->
            
      <div
         class=\\"mj-column-per-100 mj-outlook-group-fix\\" style=\\"font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"
      >
        
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" width=\\"100%\\"
      >
        <tbody>
          <tr>
            <td  style=\\"vertical-align:top;padding:0px;\\">
              
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"\\" width=\\"100%\\"
      >
        
            <tr>
              <td
                 align=\\"center\\" style=\\"font-size:0px;padding:16px 0;word-break:break-word;\\"
              >
                
      <div
         style=\\"font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:14px;font-weight:300;line-height:150%;text-align:center;color:#687C94;\\"
      >If you didn't ask for a password request, you can ignore this email. Your password will not be changed.</div>
    
              </td>
            </tr>
          
      </table>
    
            </td>
          </tr>
        </tbody>
      </table>
    
      </div>
    
          <!--[if mso | IE]>
            </td>
          
        </tr>
      
                  </table>
                <![endif]-->
              </td>
            </tr>
          </tbody>
        </table>
        
      </div>
    
      
      <!--[if mso | IE]>
          </td>
        </tr>
      </table>
      
              </td>
            </tr>
          
                  </table>
                <![endif]-->
              </td>
            </tr>
          </tbody>
        </table>
        
      </div>
    
      
      <!--[if mso | IE]>
          </td>
        </tr>
      </table>
      
      <table
         align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\"
      >
        <tr>
          <td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\">
      <![endif]-->
    
      
      <div  style=\\"margin:0px auto;max-width:600px;\\">
        
        <table
           align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"
        >
          <tbody>
            <tr>
              <td
                 style=\\"direction:ltr;font-size:0px;padding:0px;text-align:center;\\"
              >
                <!--[if mso | IE]>
                  <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
                
        <tr>
      
            <td
               class=\\"\\" style=\\"vertical-align:top;width:600px;\\"
            >
          <![endif]-->
            
      <div
         class=\\"mj-column-per-100 mj-outlook-group-fix\\" style=\\"font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"
      >
        
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" width=\\"100%\\"
      >
        <tbody>
          <tr>
            <td  style=\\"vertical-align:top;padding:0px;\\">
              
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"\\" width=\\"100%\\"
      >
        
            <tr>
              <td
                 style=\\"font-size:0px;padding:0px;word-break:break-word;\\"
              >
                
      
    <!--[if mso | IE]>
    
        <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\"><tr><td height=\\"32\\" style=\\"vertical-align:top;height:32px;\\">
      
    <![endif]-->
  
      <div
         style=\\"height:32px;\\"
      >
        &nbsp;
      </div>
      
    <!--[if mso | IE]>
    
        </td></tr></table>
      
    <![endif]-->
  
    
              </td>
            </tr>
          
      </table>
    
            </td>
          </tr>
        </tbody>
      </table>
    
      </div>
    
          <!--[if mso | IE]>
            </td>
          
        </tr>
      
                  </table>
                <![endif]-->
              </td>
            </tr>
          </tbody>
        </table>
        
      </div>
    
      
      <!--[if mso | IE]>
          </td>
        </tr>
      </table>
      <![endif]-->
    
    <!-- FOOTER -->
      
      <!--[if mso | IE]>
      <table
         align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\"
      >
        <tr>
          <td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\">
      <![endif]-->
    
      
      <div  style=\\"margin:0px auto;max-width:600px;\\">
        
        <table
           align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"
        >
          <tbody>
            <tr>
              <td
                 style=\\"direction:ltr;font-size:0px;padding:0px;text-align:center;\\"
              >
                <!--[if mso | IE]>
                  <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
                
            <tr>
              <td
                 class=\\"\\" width=\\"600px\\"
              >
          
      <table
         align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\"
      >
        <tr>
          <td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\">
      <![endif]-->
    
      
      <div  style=\\"margin:0px auto;max-width:600px;\\">
        
        <table
           align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"
        >
          <tbody>
            <tr>
              <td
                 style=\\"direction:ltr;font-size:0px;padding:0px;text-align:center;\\"
              >
                <!--[if mso | IE]>
                  <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
                
        <tr>
      
            <td
               class=\\"\\" style=\\"vertical-align:top;width:600px;\\"
            >
          <![endif]-->
            
      <div
         class=\\"mj-column-per-100 mj-outlook-group-fix\\" style=\\"font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"
      >
        
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" width=\\"100%\\"
      >
        <tbody>
          <tr>
            <td  style=\\"vertical-align:top;padding:8px 0;\\">
              
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"\\" width=\\"100%\\"
      >
        
            <tr>
              <td
                 align=\\"center\\" style=\\"font-size:0px;padding:0px;word-break:break-word;\\"
              >
                
      
     <!--[if mso | IE]>
      <table
         align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\"
      >
        <tr>
      
              <td>
            <![endif]-->
              <table
                 align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"float:none;display:inline-table;\\"
              >
                
      <tr
        
      >
        <td  style=\\"padding:0 12px;\\">
          <table
             border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"background:#687C94;border-radius:3px;width:24px;\\"
          >
            <tr>
              <td  style=\\"font-size:0;height:24px;vertical-align:middle;width:24px;\\">
                <a  href=\\"https://la.mk\\" target=\\"_blank\\">
                    <img
                       height=\\"24\\" src=\\"https://www.mailjet.com/images/theme/v1/icons/ico-social/web.png\\" style=\\"border-radius:3px;display:block;\\" width=\\"24\\"
                    />
                  </a>
                </td>
              </tr>
          </table>
        </td>
        
      </tr>
    
              </table>
            <!--[if mso | IE]>
              </td>
            
              <td>
            <![endif]-->
              <table
                 align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"float:none;display:inline-table;\\"
              >
                
      <tr
        
      >
        <td  style=\\"padding:0 12px;\\">
          <table
             border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"background:#687C94;border-radius:3px;width:24px;\\"
          >
            <tr>
              <td  style=\\"font-size:0;height:24px;vertical-align:middle;width:24px;\\">
                <a  href=\\"https://www.facebook.com/sharer/sharer.php?u=https://facebook.com/social.la.mk\\" target=\\"_blank\\">
                    <img
                       height=\\"24\\" src=\\"https://www.mailjet.com/images/theme/v1/icons/ico-social/facebook.png\\" style=\\"border-radius:3px;display:block;\\" width=\\"24\\"
                    />
                  </a>
                </td>
              </tr>
          </table>
        </td>
        
      </tr>
    
              </table>
            <!--[if mso | IE]>
              </td>
            
              <td>
            <![endif]-->
              <table
                 align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"float:none;display:inline-table;\\"
              >
                
      <tr
        
      >
        <td  style=\\"padding:0 12px;\\">
          <table
             border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"background:#687C94;border-radius:3px;width:24px;\\"
          >
            <tr>
              <td  style=\\"font-size:0;height:24px;vertical-align:middle;width:24px;\\">
                <a  href=\\"https://instagram.com/_la.mk\\" target=\\"_blank\\">
                    <img
                       height=\\"24\\" src=\\"https://www.mailjet.com/images/theme/v1/icons/ico-social/instagram.png\\" style=\\"border-radius:3px;display:block;\\" width=\\"24\\"
                    />
                  </a>
                </td>
              </tr>
          </table>
        </td>
        
      </tr>
    
              </table>
            <!--[if mso | IE]>
              </td>
            
          </tr>
        </table>
      <![endif]-->
    
    
              </td>
            </tr>
          
      </table>
    
            </td>
          </tr>
        </tbody>
      </table>
    
      </div>
    
          <!--[if mso | IE]>
            </td>
          
        </tr>
      
                  </table>
                <![endif]-->
              </td>
            </tr>
          </tbody>
        </table>
        
      </div>
    
      
      <!--[if mso | IE]>
          </td>
        </tr>
      </table>
      
              </td>
            </tr>
          
            <tr>
              <td
                 class=\\"\\" width=\\"600px\\"
              >
          
      <table
         align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" class=\\"\\" style=\\"width:600px;\\" width=\\"600\\"
      >
        <tr>
          <td style=\\"line-height:0px;font-size:0px;mso-line-height-rule:exactly;\\">
      <![endif]-->
    
      
      <div  style=\\"margin:0px auto;max-width:600px;\\">
        
        <table
           align=\\"center\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"width:100%;\\"
        >
          <tbody>
            <tr>
              <td
                 style=\\"direction:ltr;font-size:0px;padding:0px;text-align:center;\\"
              >
                <!--[if mso | IE]>
                  <table role=\\"presentation\\" border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
                
        <tr>
      
            <td
               class=\\"\\" style=\\"vertical-align:top;width:600px;\\"
            >
          <![endif]-->
            
      <div
         class=\\"mj-column-per-100 mj-outlook-group-fix\\" style=\\"font-size:0px;text-align:left;direction:ltr;display:inline-block;vertical-align:top;width:100%;\\"
      >
        
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" width=\\"100%\\"
      >
        <tbody>
          <tr>
            <td  style=\\"vertical-align:top;padding:0px;\\">
              
      <table
         border=\\"0\\" cellpadding=\\"0\\" cellspacing=\\"0\\" role=\\"presentation\\" style=\\"\\" width=\\"100%\\"
      >
        
            <tr>
              <td
                 align=\\"center\\" style=\\"font-size:0px;padding:8px 0;word-break:break-word;\\"
              >
                
      <div
         style=\\"font-family:Ubuntu, Helvetica, Arial, sans-serif;font-size:14px;font-weight:300;line-height:150%;text-align:center;color:#687C94;\\"
      >la.mk © 2020 All rights reserved</div>
    
              </td>
            </tr>
          
      </table>
    
            </td>
          </tr>
        </tbody>
      </table>
    
      </div>
    
          <!--[if mso | IE]>
            </td>
          
        </tr>
      
                  </table>
                <![endif]-->
              </td>
            </tr>
          </tbody>
        </table>
        
      </div>
    
      
      <!--[if mso | IE]>
          </td>
        </tr>
      </table>
      
              </td>
            </tr>
          
                  </table>
                <![endif]-->
              </td>
            </tr>
          </tbody>
        </table>
        
      </div>
    
      
      <!--[if mso | IE]>
          </td>
        </tr>
      </table>
      <![endif]-->
    
    
      </div>
    
      </body>
    </html>
  "
`;
